msg/SemanticClass.msg

```diff
@@ -0,0 +1,10 @@
+# A key value pair that maps an integer class_id to a string class label.
+# The class_id should be interpreted as the pixel value corresponding to a 
```

Comment:
There's trailing whitespace in this line which makes the tests fail. Also both files are missing a newline at the end of the file.